src/helm/benchmark/presentation/summarize.py

```diff
@@ -268,25 +269,37 @@ def filter_runs_by_visibility(self, runs: List[Run], group: RunGroup) -> List[Ru
             included = True
             if group.visibility == THIS_GROUP_ONLY:  # don't include the canonical runs when looking at, say, ablations
                 included = False
+
             for run_group_name in run.run_spec.groups:  # go through the groups of the run to determine visibility
                 if run_group_name not in self.schema.name_to_run_group:
                     hlog(
                         f"WARNING: group {run_group_name} mentioned in run spec {run.run_spec.name} "
                         f"but undefined in {SCHEMA_YAML_FILENAME}, skipping"
                     )
                     continue
+
                 run_group = self.schema.name_to_run_group[run_group_name]
                 if run_group.visibility == NO_GROUPS:  # this run should never be visible
                     included = False
                     break
-                if run_group.visibility == THIS_GROUP_ONLY:  # this run is part of a group with partial visibility
+                elif run_group.visibility == THIS_GROUP_ONLY:  # this run is part of a group with partial visibility
                     if run_group.name == group.name:  # if this is the exact group we are visualizing, include for sure
                         included = True
                         break
                     else:  # we won't visualize unless we hit exactly the group with partial visibility
                         included = False
+                elif run_group.visibility == ALL_GROUPS:
```

Comment:
@dtsip I wanted to double check if adding this logic is okay.

Reply:
hm, this would actually mess with the visualization logic and cause ablations to show up together with the core runs...

What is the bug we are fixing here? Not sure I get it from the examples above....

(Existing logic: Let's say we have two IMDB run. One is the core run with `groups = ["imdb" (visibility: ALL_GROUPS)]` which means it gets visualized when we look at the group "imdb" but also when we look at ablations. The other run is with `groups = ["imdb" (visibility: ALL_GROUPS), "in-context-ablations" (visibility: THIS_GROUP_ONLY]` which means that it will get visualized only when we look at `in-context-ablations`, since the restricted visibility of the `in-context-ablations` group prevents it from showing up in the `imdb` results. The proposed changed would cause the second group to be included everywhere.)